src/benchmark.cc

```diff
@@ -230,7 +230,7 @@ void State::PauseTiming() {
       auto name = name_and_measurement.first;
       auto measurement = name_and_measurement.second;
       BM_CHECK_EQ(std::fpclassify(double{counters[name]}), FP_ZERO);
```

Comment:
I don't believe this assertion remains valid after this change.

Reply:
thanks for the catch - in a debug build it trips, indeed.